### ISSUE TITLE
What type of ban is it?

### ISSUE THREAD

**Issue by ErrorDodo** [IstAuchEgal/Steam-Ban-Check#1]
# Problem

Currently your iteration of ban checking will return any ban, how is the end user meant to know what type of ban, is it a game ban or a VAC ban?

# Solution

There is a really good python library already for a [Steam wrapper](https://github.com/ValvePython/steam).

There are [examples](https://github.com/ValvePython/steam/tree/master/recipes) of how to log into the accounts with the python library.

---

Using this library you won't need an api call to a webservice (although the library simplifies it if needed).

Checking for ban can prove to be quite painful if you are not used to using Steam GameCoordinator (If you're attempting to check bans in games I.E CSGO, Dota 2).

The Pseudo code of checking for a ban for csgo would look something like this:
```
Log into steam account with values provided

(Python library could have an event that "subscribes" to a vac ban checker every time an account is successfully logged in)

Check the account for VAC bans

If there is a VAC ban on the account, and the VAC ban is tied to CSGO (appid:  730) return "VAC Banned In CSGO"

Otherwise return "Not VAC Banned In CSGO"
```

---

## Still a bit confused?

I have some good examples for you.

My friend created a CSGO VAC ban/Cooldown checker in two different languages but they should port to python quite easily.

### The c# one
[CSGO Ban Checker in c#](https://github.com/yeahmaybeno/csgo-banchecker)

### The javascript one
[CSGO Ban Checker in javascript](https://github.com/ErrorDodo/Steam_NonPrime_Checker)

**Comment by IstAuchEgal** [comment]
Yes, sadly you cant just see the exact game like this, but you CAN see if its a game, vac or community ban also how old it is
here is an example output: 
`{"players":[{"SteamId":"76561198087086399","CommunityBanned":false,"VACBanned":true,"NumberOfVACBans":1,"DaysSinceLastBan":1313,"NumberOfGameBans":0,"EconomyBan":"none"}]}`
Originaly i wanted to avoid loging in to each acc but now I think its impossible without

**Comment by ErrorDodo** [comment]
Hmm I believe logging into the accounts will be more of what you are after, I remember reading you have all the accounts in a spreadsheet?

I'm not sure the format of your spreadsheet but I would just log into the account and write the information to a new column on the spreadsheet
